django/views/templates/technical_500.html

```diff
@@ -195,11 +199,11 @@ <h3>{{ template_info.message|force_escape }}</h3>
       {% if template_info.bottom != template_info.total %} cut-bottom{% endif %}">
    {% for source_line in template_info.source_lines %}
    {% if source_line.0 == template_info.line %}
-   <tr class="error"><th>{{ source_line.0 }}</th>
+   <tr class="error"><th scope="row">{{ source_line.0 }}</th>
      <td>{{ template_info.before }}<span class="specific">{{ template_info.during }}</span>{{ template_info.after }}</td>
    </tr>
    {% else %}
-      <tr><th>{{ source_line.0 }}</th>
+      <tr><th scope="row">{{ source_line.0 }}</th>
```

Comment:
I don't think that the line number in templates can be treated as a header. IMO, we should change it to `<td>`.

Reply:
That's a good point, and I agree, but that would leave the table in a heading-less state. Neither situation is preferable.

Changing anything more significant about this code would be out of the scope of this PR. What would you recommend we do? Maybe @thibaudcolas can chime in on this one?

Reply:
As far as I'm aware, tables without headings are better than tables with incorrect ones.

Reply:
We could also display it in a single cell:
```html
<tr><td>{{ source_line.0 }}: {{ source_line.1 }}</td></tr>
```
or use `<ol start="..."><li>...</li></ol>` instead (like for tracebacks).

Reply:
I'm thinking about this but don't have a good answer yet.

A single-cell approach would turn this table into a list, which is not semantically correct.

(Solution 1) We could add a header row to this table with a heading cell for "Path" and "Error".

(Solution 2) If not, switching to an ordered list would be good, too.
However, the traceback feature is inaccessible, making me reluctant to introduce it now. (Fixing that would be a separate ticket)

Reply:
![Screenshot 2024-01-29 at 11 29 07](https://github.com/django/django/assets/2210890/cac4c081-7aba-4834-81a0-d909d6f44e9b)
Quick mock-up of solution 1. 

I'll leave it with this screenshot for now. It does not feel good to me to replace the `<th>` with a `<td>`, but I also don't know enough to decide the best course of action.

Since Thibaud proposed this change and is part of the accessibility team, I would also like to hear his opinion.

If you rather go ahead: say the word, and I'll make the change so you can merge it.

Reply:
Convinced.

Reply:
Alright, I'm leaving it as it is now, then.

Reply:
Yes, please do.